asset-metadata/src/static/style.css

```diff
@@ -34,7 +34,7 @@ body {
 @media (min-width: 1440px) {
   .metadata {
     left: 60px;
-    top: 180px;
+    top: 120px;
```

Comment:
Unintended addition?

Reply:
It's intentional. The left box goes down a lot when it has more content, so just giving it some space.